src/js/msp/MSPHelper.js

```diff
@@ -1458,6 +1460,9 @@ MspHelper.prototype.crunch = function(code) {
                 .push8(SENSOR_ALIGNMENT.align_mag);
             break;
         case MSPCodes.MSP_SET_ADVANCED_CONFIG:
+            if (PID_ADVANCED_CONFIG.motor_pwm_rate == 0 || (semver.gte(CONFIG.apiVersion, "1.24.0") && PID_ADVANCED_CONFIG.digitalIdlePercent == 0)) {
```

Comment:
Again, this won't work, as the sender of `MSP_SET_ADVANCED_CONFIG` expects this to be sent irrespective of values.